src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -146,6 +157,7 @@ export class FileExplorerComponent implements OnInit {
     });
     this.selectedFiles = newSelection;
     this.countFiles();
+    this.sumSize();
```

Comment:
Instead of duplicating the code that calls these functions, there should be a single function that updates both totals that's called from each method.

Or, better yet, the code should observe `selectedFiles` and update both counts if the selected files changes.